manager/manager.go

```diff
@@ -49,6 +49,9 @@ type Config struct {
 	// ProtoAddr fields will be used to create listeners otherwise.
 	ProtoListener map[string]net.Listener
 
+	// AdvertiseAddr is a map of addresses to advertise, by protocol.
+	AdvertiseAddr map[string]string
```

Comment:
I imagine if move to a model where we support multiple listeners per protocol, we'll update the data structures accordingly.

This does not seem relevant to unix sockets.